src/read_csv.py

```diff
@@ -5,32 +5,38 @@
 import csv
 import sys
 import pandas as pd
+import os
 
+PATH = sys.path[0]
 
 def transform_csv_from_bad_format_to_better_format(data_list, new_csv_name):
     """
     Takes in dataList, which represents the original CSV, and writes a new
     CSV file that has mapped all the point with their name.
     """
     # Find the first empty row and create a dictionary to map the points to their names:
-    split = 0
+    split = 1
     point_to_name = {}
-    while split < len(data_list) and data_list[split][0] != "":
+    while "Point_" in data_list[split][0]:
         point_to_name[(data_list[split][0])[:-1]] = data_list[split][1]
         split += 1
-    
+
     # Write the new data to a CSV:
     with open(new_csv_name, 'w') as f:
         writer = csv.writer(f)
         
         # Construct and write the first line
         first_line = [data_list[split + 1][0], data_list[split + 1][1]]
-        for j in range(2, len(data_list[split + 1]), 1):
-            first_line.append(point_to_name[data_list[split + 1][j]])
         writer.writerow(first_line)
-        
+        writer.writerow("")
+
+        first_line = ["Date","Time"]
```

Comment:
Does this not end up writing Date and Time twice? Since I believe that is what line 30 is doing as well, though I could be mistaken.

Reply:
Line 30 I think is adding the entire date range, so it would be something along the lines of 

Date Range: | 7/8/2014 00:00:00 - 11/12/2014 23:59:59
-- | --

Reply:
Ah makes sense